src/as-validator-issue-tag.h

```diff
@@ -685,6 +685,11 @@ AsValidatorIssueTag as_validator_issue_tag_list[] =  {
 	  N_("The value set as release type is invalid."),
 	},
 
+	{ "release-property-missing",
+	  AS_ISSUE_SEVERITY_INFO,
```

Comment:
This will have at least `WARNING` severity, as it violates the specification and should make the validation fail. I would say that a missing version would also definitely be an `ERROR` severity, as it will actually make the release data unusable on most application (I think libappstream will even refuse to read such a release).
If you want to make both takes errors, that would be fine with me too though.